helpers/helpers.go

```diff
@@ -398,21 +398,31 @@ func CheckTokenExpiration(accessToken string) error {
 	}
 
 	switch untilExp := time.Until(expiration); {
-	case untilExp < 0:
-		return fmt.Errorf("the provided access token has expired, please renew it")
-	case untilExp > 0 && untilExp < 24*time.Hour:
-		fmt.Fprintln(
+	case untilExp >= 1*time.Minute && untilExp < 2*time.Hour:
+		hourString := ""
+		if int(untilExp.Hours()) != 0 {
+			hourString = "1 hour and "
+		}
+		fmt.Fprintf(
```

Comment:
I'm not a fan of this style of coding.
This could be handled by another case in the switch:

```suggestion
	case untilExp >= 1*time.Minute && untilExp <= 1*time.Hour:
		fmt.Fprintf(
			os.Stderr,
			"WARNING! The provided access token expires in only %d minutes.\n", 
			int(untilExp.Minutes())-(int(untilExp.Hours())*60),
		)

	case untilExp > 1*time.Hour && untilExp < 2*time.Hour:
```

Reply:
Probably I put too much sauce here. Changed it to another case as suggested.